index.d.ts

```diff
@@ -4,7 +4,7 @@
 /**
  * Container for all registered metrics
  */
-export interface register {
+export interface Registry {
```

Comment:
In `index.js` `Registry` is added as extra property:
```js 
exports.Registry = require('./lib/registry');
```
However in this file old `register` is removed and replaced with `Registry`.